include/ofi_util.h

```diff
@@ -329,6 +329,18 @@ typedef void (*ofi_ep_cntr_inc_func)(struct util_ep *);
 extern ofi_ep_cntr_inc_func ofi_ep_tx_cntr_inc_funcs[ofi_op_last];
 extern ofi_ep_cntr_inc_func ofi_ep_rx_cntr_inc_funcs[ofi_op_last];
 
+static inline void ofi_ep_tx_cntr_inc_func(struct util_ep *ep, uint8_t op)
+{
+	assert(op < ofi_op_last);
+	ofi_ep_tx_cntr_inc_funcs[op](ep);
+}
+
+static inline void ofi_ep_rx_cntr_inc_func(struct util_ep *ep, uint8_t op)
+{
+	assert(op < ofi_op_last);
```

Comment:
Should this be renamed to ofi_op_max? ofi_op_last seems to imply last (previous) op.

Reply:
I agree max is more common for enum ranges.  This could go in a separate PR